specification/prescriptions-for-patients.yaml

```diff
@@ -220,10 +220,11 @@ components:
       name: X-Correlation-ID
       required: false
       description: |
-        An optional ID which you can use to track transactions across multiple systems. It can have any value, but we recommend avoiding `.` characters.
+        An optional ID which you can use to track transactions across multiple systems. If included its value must be a valid GUID.
         Mirrored back in a response header.
```

Comment:
Could you please add a comma to the sentence:
If included, its value must be a valid GUID.